control-plane/subcommand/gateway-resources/command.go

```diff
@@ -331,37 +359,104 @@ func (c *Command) loadConfig() error {
 	return nil
 }
 
+// loadMeshConfigs reads and loads the mesh configs from `/consul/config/config.yaml`, if this file does not exist nothing is done
```

Comment:
Related to above comments, maybe this becomes loadConfig() and the legacy becomes loadConfigv1() or stays as  loadAPIGWResourceConfig()? (Not sold on those names).
Eventually everything will come as one config file.

Reply:
yeah, I'm down for either, it might make sense to keep the existing as `loadAPIGWResourceConfig` since that's really the only config it's loading but open to suggestions

Reply:
make sense to me. this part is internal, so we can change it in the future. wanted to make sure the user-facing flags were updated first and foremost

Reply:
We can leave for now, but that comment still needs to be updated.

Reply:
ugh, stupid case sensitive find and replaces